api/modules/zomato/views.py

```diff
@@ -0,0 +1,52 @@
+import requests
+from rest_framework.decorators import api_view
+import requests_cache
+from datetime import timedelta
+from rest_framework import status
+from rest_framework.response import Response
+from api.modules.zomato.constants import BASE_URL, header
```

Comment:
Change module name to `hyperlocal/food` instead of `zomato`
This gives us flexibility to use any other API in future without changing the module interface